MastodonSDK/Sources/MastodonCore/Extension/MastodonSDK/Mastodon+Entity+StatusEdit.swift

```diff
@@ -0,0 +1,15 @@
+// Copyright © 2023 Mastodon gGmbH. All rights reserved.
+
+import Foundation
+import MastodonSDK
+
+extension Mastodon.Entity.StatusEdit: Hashable, Equatable {
```

Comment:
The reason I did this is because it's aligned to other extensions of those entities, but I agree that this spreads code over different places and makes discoverability worse. I'll move it to the original file but leave it in a distinct extension to keep the model class simpler.